src/verifier.cc

```diff
@@ -254,12 +261,15 @@ Result Verifier::Probe(const ProbeCommand* command,
         (command->IsRGBA() ? ", " + std::to_string(command->GetA() * 255) +
                                  "\n  Actual RGBA: "
                            : "\n  Actual RGB: ") +
-        std::to_string(static_cast<int>(p[stride * first_invalid_i])) + ", " +
-        std::to_string(static_cast<int>(p[stride * first_invalid_i + 1])) +
+        std::to_string(static_cast<int>(p[texel_stride * first_invalid_i])) +
```

Comment:
If we store texel_stride * first_invalid_i somewhere and reuse it in these messages, does this format a bit nicer?

Reply:
Yep.  Looks better when we fold more right into the p base pointer.